broken-links-script/cypress/e2e/link-checker.cy.js

```diff
@@ -136,17 +145,35 @@ describe('Link and Routing Validation - Individual URL Checks', () => {
         }
       }
       else if (isApiPage) {
-        cy.visit(url, { timeout: 50000 });
+        cy.visit(url, { timeout: 20000 });
         if (fragment) {
-          cy.get(`[id="${fragment}"]`, { timeout: 10000 }).should('exist');
+          cy.get(`[id="${fragment}"]`, { timeout: 20000 }).should('exist');
+        }
+      }
+      else if (isGithubBlobLine) {
+        const baseUrl = url.split('#')[0];
+        const match = url.match(/#L(\d+)/);
+        const lineNumber = match ? match[1] : null;
+        cy.request({ url: baseUrl, failOnStatusCode: false }).then((res) => {
+          expect(res.status, `GitHub blob file should exist: ${baseUrl}`)
+            .to.be.oneOf([200, 301, 302]);
+        });
+        if (lineNumber) {
+          cy.visit(baseUrl, { timeout: 30000 });
+          const selector = `#L${lineNumber}, #LC${lineNumber}`;
+          cy.get('body').then(($body) => {
+            if ($body.find(selector).length === 0) {
+              throw new Error(`Line ${lineNumber} does NOT exist in ${baseUrl}`);
+            }
+          });
```

Comment:
Isn't it the same as:
```suggestion
          const selector = `#L${lineNumber}, #LC${lineNumber}`;
          cy.get(selector).should('exist');
```
Do you need either of the selectors to exist or both must exist?